dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -2411,10 +2411,10 @@ def  put_participant_in_section(student, teacher, script)
         assert_response :success
       end
 
-      it '/courses/:course_course_name/units/:unit_position/lessons/:lesson_position/levels/:position does not redirect' do
+      it '/courses/:course_course_name/units/:unit_position/lessons/:lesson_position/levels/:position does redirect' do
         sign_in user
         get :show, params: {course_course_name: course.name, unit_position: unit_position, lesson_position: lesson_position, id: level.position}
-        assert_response :success
+        assert_redirected_to "/s/#{unit.name}/lessons/#{lesson_position}/levels/#{level.position}"
```

Comment:
can you help me understand why this behavior is changing?

Reply:
I decided to make it so the new nested URLs would redirect back to /s/ URLs if the 'modularity' experiment is off. This is just in case we launch modularity and discover some critical issue with it. We will redirect any users currently clicking around in the nested URLs back the old, stable /s/ URLs.